src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/BalanceSheetController.java

```diff
@@ -27,7 +28,12 @@ public String getBalanceSheet(@PathVariable String transactionId,
                                   @PathVariable String companyAccountsId,
                                   Model model) {
 
-        model.addAttribute("balanceSheet", balanceSheetService.getBalanceSheet(transactionId, companyAccountsId));
+        try {
+            model.addAttribute("balanceSheet", balanceSheetService.getBalanceSheet(transactionId, companyAccountsId));
+        } catch (ApiErrorResponseException e) {
+            // TODO: Handle with exception handler once implemented
+            e.printStackTrace();
```

Comment:
This should be using structured logging (if it gets missed in a future PR it won't be logging in the correct format).

Reply:
Used the Structured Logging library for `LOGGER.error()` in place of printing the stack trace. Until Api validation is enforced, we can assume these errors are truly exceptional so the `error` log level would be the correct choice as opposed to `info` or similar.